akka-actor-typed-tests/src/test/scala/docs/akka/typed/StashDocSpec.scala

```diff
@@ -4,90 +4,107 @@
 
 package docs.akka.typed
 
-import scala.concurrent.Future
-import scala.util.Failure
-import scala.util.Success
-
-import akka.Done
 import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
-import akka.actor.typed.ActorRef
-import akka.actor.typed.Behavior
-import akka.actor.typed.scaladsl.Behaviors
+
 import org.scalatest.WordSpecLike
 
 object StashDocSpec {
   // #stashing
+  import scala.concurrent.Future
+  import scala.util.Failure
+  import scala.util.Success
+
+  import akka.Done
+  import akka.actor.typed.ActorRef
+  import akka.actor.typed.Behavior
+  import akka.actor.typed.scaladsl.ActorContext
+  import akka.actor.typed.scaladsl.Behaviors
+  import akka.actor.typed.scaladsl.StashBuffer
+
   trait DB {
     def save(id: String, value: String): Future[Done]
     def load(id: String): Future[String]
   }
 
   object DataAccess {
-    trait Command
+    sealed trait Command
     final case class Save(value: String, replyTo: ActorRef[Done]) extends Command
     final case class Get(replyTo: ActorRef[String]) extends Command
     private final case class InitialState(value: String) extends Command
     private final case object SaveSuccess extends Command
     private final case class DBError(cause: Throwable) extends Command
```

Comment:
Out of scope but not a command.

Reply:
The naming convention for incoming messages are `Command`. DBError i via pipeToSelf, but still an incoming message. See:
* https://doc.akka.io/docs/akka/snapshot/typed/style-guide.html#additional-naming-conventions
* https://doc.akka.io/docs/akka/snapshot/typed/style-guide.html#public-versus-private-messages